tutorials/polkadot-sdk/build-a-blockchain/upgrade-a-running-network.md

```diff
@@ -0,0 +1,262 @@
+
+- Increase the `spec_version`
+- Add the Utility pallet
+- Increase the minimum balance for network accounts
+
+By the end of this tutorial, you’ll have the skills to upgrade the runtime and submit a transaction to deploy the modified runtime on a live network.
+
+## Prerequisites
+
+Before starting this tutorial, ensure you meet the following requirements:
+
+- Installed and configured Rust on your system. Refer to the [Installation]() guide for detailed instructions on installing Rust and setting up your development environment
+- Completed the [Build a Local Blockchain]() tutorial and have the [Polkadot-SDK Solochain Template](https://github.com/paritytech/polkadot-sdk-solochain-template){target=\_blank}installed on your machine
+- Reviewed the [Add a Pallet to the Runtime]() guide
+
+## Start the Node
+
+To demonstrate how to update a running node, you first need to start the local node with the current runtime.
+
+1. Navigate to the root directory where you compiled the Polkadot-SDK Solochain Template
+   
+2. Start the local node in development mode by running the following command:
+    ```bash
+    ./target/release/solochain-template-node --dev
+    ```
+
+    !!!note
+       Keep the node running throughout this tutorial. You can modify and re-compile the runtime without stopping or restarting the node.
+
+3. Connect to your node using the same steps outlined in the [Interact with the Node]() section
+
+    Once connected, you’ll notice the node template is using the default version, `100`, displayed in the upper left.
+
+    ![](/images/tutorials/polkadot-sdk/build-a-blockchain/upgrade-a-running-network/upgrade-a-running-network-1.webp)
+
+## Modify the Runtime
+
+### Add the Utility Pallet to the Dependencies
+
+1. Open the `runtime/Cargo.toml` file and locate the `[dependencies]` section. Add the Utility pallet by inserting the following line:
+
+    ```toml
+    pallet-utility = { version = "37.0.0", default-features = false}
+    ```
+
+    Your `[dependencies]` section should now look something like this:
+
+    ```toml
+    [dependencies]
+    codec = { features = ["derive"], workspace = true }
+    scale-info = { features = ["derive", "serde"], workspace = true }
+    frame-support = { features = ["experimental"], workspace = true }
+    ...
+    pallet-utility = { version = "37.0.0", default-features = false }
+    ```
+
+2. In the `[features]` section, add the Utility pallet to the `std` feature list by including:
+    ```toml
+    [features]
+    default = ["std"]
+    std = [
+        "codec/std",
+        "scale-info/std",
+        "frame-executive/std",
+        ...
+        "pallet-utility/std",
+    ]
+    ```
+
+3. Save the changes and close the `Cargo.toml` file
+
+### Update the Runtime Configuration
+
+To implement the necessary changes in your runtime, you'll need to modify the `runtime/src/lib.rs` file.
+
+#### Configure the Utility Pallet
+   
+1. Implement the Config trait for the Utility pallet:
```

Comment:
```suggestion
1. Implement the Config trait for the Utility pallet:
```
So...generally we want to use configuration rather than config unless that is the name of a thing where it's used like `Config` trait. Please update according to whichever is true in this situation.

Reply:
Yeah, here it's most likely `Config` - added a suggestion

Reply:
Fixed